packages/lesswrong/server/register-mui-styles/scripts/gen-mui-registration.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# See note in ../index.ts.
```

Comment:
This script is terrible and not very reliable. But falling back to adding components manually is always an option.